pytest_watch/watcher.py

```diff
@@ -117,10 +119,18 @@ def _reduce_events(events):
     return filtered
 
 
+def _bright(arg):
+    return STYLE_BRIGHT + arg + Style.RESET_ALL
+
+
+def _highlight(arg):
+    return STYLE_HIGHLIGHT + arg + Style.RESET_ALL
+
+
 def _show_summary(argv, events, verbose=False):
     command = ' '.join(argv)
-    bright = lambda arg: STYLE_BRIGHT + arg + Style.RESET_ALL
```

Comment:
__________ PEP8-check __________
pytest_watch/watcher.py:124:5: E731 do not assign a lambda expression, use a def
bright = lambda arg: STYLE_BRIGHT + arg + Style.RESET_ALL
^

Reply:
just for pep8 purposes